sycl/test-e2e/Regression/optimization_level_debug_info_specopt.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} %debug_option -Ofast -o %t.out
+// RUN: %{build} %debug_option -O3 -ffast-math -o %t.out
```

Comment:
@zahiraam can you confirm this is ok

Reply:
The `-Ofast` should be replaced with `ffp-model=aggressive` instead.

Reply:
I think you meant `-ffp-model=fast`, there is NO `aggressive`.

Reply:
There should be. See https://github.com/llvm/llvm-project/pull/100453
Otherwise `ffp-model=fast`.

Reply:
https://github.com/llvm/llvm-project/pull/100453 is NOT yet pulldown in this PR.